docs/manual/java/gettingstarted/index.toc

```diff
@@ -2,10 +2,8 @@ IntroGetStarted:Introduction
 JavaPrereqs:Lagom Java prerequisites;next=GettingStartedMaven,GettingStartedSbt
 GettingStartedMaven:Creating and running Hello World with Maven;next=UnderstandHello
 GettingStartedSbt:Creating and running Hello World with sbt;next=UnderstandHello
-UnderstandHello:Understanding Hello World;next=MavenIDEs,EclipseSbt,IntellijSbt
-MavenIDEs:Integrating Maven projects with an IDE;next=LagomExamples
-EclipseSbt:Integrating sbt with Eclipse;next=LagomExamples
-IntellijSbt:Integrating sbt with IntelliJ;next=LagomExamples
+UnderstandHello:Understanding Hello World
 LagomExamples:Lagom examples
+WhatsNext:What's Next?
```

Comment:
I did some surveys of the documentation of a few projects I'm familiar with, here's Spring:

Websphere AS - https://www.ibm.com/support/knowledgecenter/en/SSEQTP_8.5.5/as_ditamaps/was855_welcome_base_dist_iseries.html - consistently sentence case

.NET - https://msdn.microsoft.com/en-us/library/w0x726c2(v=vs.110).aspx - consistently title case (note though that prepositions and conjunctions are lowercase)

Spring - http://docs.spring.io/spring/docs/current/spring-framework-reference/html/ - inconsistent, but once you get past the overview and what's new sections, seems to rather consistently use sentence case.

Hibernate - http://docs.jboss.org/hibernate/orm/5.2/userguide/html_single/Hibernate_User_Guide.html - Inconsistent, but seems to mostly use sentence case (note that there are a lot of proper nouns in the titles, capitalised proper nouns is still sentence case).


Python - https://docs.python.org/3/reference/index.html - consistently sentence case, but https://docs.python.org/3/library/index.html is inconsistent.

So, when it comes to large enterprise tech companies (Microsoft and IBM) who likely have large teams of tech writers writing their documentation, they are always consistent, but we have examples of both sentence case and title case.  When it comes to open source projects, there's little to no consistency, but this small sample seems to prefer sentence case.